packages/components/psammead-podcast-promo/src/__snapshots__/index.test.jsx.snap

```diff
@@ -208,8 +208,8 @@ exports[`Podcast Promo Examples basic example 1`] = `
 
 .emotion-2 > svg {
   margin-left: 0;
-  width: 22px;
-  height: 22px;
+  width: 1.375rem;
+  height: 1.375rem;
   fill: currentColor;
   position: relative;
   bottom: 5px;
```

Comment:
Should this be a pixel value? What's this for?

Reply:
These are the podcast icon svg's width & height. REM scales based on the base html font size, whereas hard-coded pixel values don't. This change makes the SVG scale as the user changed their browser font size

Reply:
These are just visually offsetting the SVG slightly to get its alignment correct. I've changed them to REM so that they scale with browser font size changes